src/libpriv/rpmostree-postprocess.cxx

```diff
@@ -1266,7 +1267,22 @@ filter_xattrs_impl (OstreeRepo     *repo,
           const char *validkey = accepted_xattrs[i];
           const char *attrkey = g_variant_get_bytestring (key);
           if (g_str_equal (validkey, attrkey))
-            g_variant_builder_add (&builder, "(@ay@ay)", key, value);
+            {
+              g_variant_builder_add (&builder, "(@ay@ay)", key, value);
+              continue;
+            }
+        }
+      if (tdata->xattr_translate != NULL)
```

Comment:
Should this come before handling of `accepted_xattrs`? That way, you can translate towards something in the hardcoded list.